routes/volunteer_hours/volunteer_hours.js

```diff
@@ -6,6 +6,50 @@ const hoursRouter = express();
 
 hoursRouter.use(express.json());
 
+// createSubmittedHours
+hoursRouter.post('/create', async (req, res) => {
+  try {
+    const { userId, eventId, startDatetime, endDatetime, approved, notes } = req.body;
+    // calculate number of hours
+    const start = new Date(startDatetime);
+    const end = new Date(endDatetime);
+    const diff = end.getTime() - start.getTime();
+    const numHours = diff / (60000 * 60);
+
+    const createdHours = await pool.query(
+      'INSERT INTO volunteer_hours(user_id, event_id, start_datetime, end_datetime, approved, num_hours, notes, submitted) VALUES($1, $2, $3, $4, $5, $6, $7, $8) RETURNING *;',
+      [userId, eventId, startDatetime, endDatetime, approved, numHours, notes, true],
+    );
+    res.status(200).json(createdHours.rows[0]);
```

Comment:
- should just send createdHours.rows rather than rows[0]. 

- please add error handling for the following cases:

1. query failed, no data returned (status 400)
2. successful query, status 200 (current line of code)